### ISSUE TITLE
Can't save port addresses

### ISSUE THREAD

**Issue by Nullifier** [j4s0n-c/trowaSoft-VCV#49]
Trying to edit default Out Port 7000, In Port 7001 to custom ones by changing the numbers and exiting out of the config window (as told in your guide). But it won't save new port names and IP Address as well. The only window that keeps changes is "namespace". 

I tried different approaches but nothing seems to help. Please let me know if there is a fix for that or maybe I am doing something wrong

VCV Rack 1.1.16
cvOSCcv  v1.0.2

**Comment by chichian** [comment]
Hi Nullifier,

Sorry for the late reply. I don't have VCV Rack on my PC right now, but if you click the "Connect" button I think it should save the configuration ports for sure (it makes sure it can connect to the IP and the ports before saving). The hiding the configuration screen is for the channel addresses in the large window on the bottom.

Let me know if it works.

**Comment by Nullifier** [comment]
Oh, man, thank you so much. I was tired of searching for any solution. And it was so simple. Thank you for coming back!

**Comment by Nullifier** [comment]
It seems that anyway I can't make it work. It just won't receive any signal from other apps. I've tested it with TouchDesigner and Notch Builder. 
It can only send OSC messages but won't receive any.

**Comment by j4s0n-c** [comment]
@Nullifier 
This quick test setup works for me, sending / receiving OSC on same machine. Do you mind sharing your intended usage? It's not really recommended for this type of smooth data stream as it gets very heavy on the CPU real quick. I tried with two LFOs from TD, and Rack crashes immediately on this little old macbook pro. Better rig can deff do more but, beware...
<img width="1680" alt="Screen Shot 2020-06-11 at 12 42 10 PM" src="https://user-images.githubusercontent.com/4063528/84432334-8a081c00-abe1-11ea-9bf2-611a78cebe6c.png">

**Comment by Nullifier** [comment]
@j4s0n-c  
So it seems my problem was solved after I removed the default "trowacv" text in "namespace". I thought it is only used for outgoing messages. I've rebuilt your setup and it works perfectly.

In my case, there was a problem now with Kinect > oscout in TD, probably did something wrong with chanel names, cvOSCcv just wouldn't receive any data. But I've found a workaround with different software (NOTCH Builder) so no need to use TD for Kinect data anymore.
Thank you guys for a great free tool! We are preparing an interactive audiovisual dance performance with all those tools and modular sounds. It wouldn't be possible without your help.
